release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -3354,6 +3356,60 @@ This section will continue to be updated over time to provide notes on enhanceme
 For any {product-title} release, always review the instructions on xref:../updating/updating_a_cluster/updating-cluster-web-console.adoc#updating-cluster-web-console[updating your cluster] properly.
 ====
 
+//4.16.39
+[id="ocp-4-16-39_{context}"]
+=== RHSA-2025:4008 - {product-title} {product-version}.39 bug fix and security update
+
+Issued: 23 April 2025
+
+{product-title} release {product-version}.39 is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2025:4008[RHSA-2025:4008] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2025:4010[RHBA-2025:4010] advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.16.39 --pullspecs
+----
+
+[id="ocp-4-16-39-bug-fixes_{context}"]
+==== Bug fixes
+
+* Previously, containers that used the SELinux `container_logreader_t` domain for the purposes of viewing container logs on a host at `/var/log` could not access logs in the `/var/log/containers` subdirectory. This issue occurred because of a missing symbolic link. With this release, a symbolic link is created for `/var/log/containers` so that containers can access the logs in `/var/log/containers`. (link:https://issues.redhat.com/browse/OCPBUGS-54343[OCPBUGS-54343]) 
```

Comment:
Maybe it's me but I do not see "54343" on the list:

![Screenshot From 2025-04-23 10-47-54](https://github.com/user-attachments/assets/fb6675d3-b73c-4f82-afde-714ad1f3fb35)

Reply:
Where did OCPBUGS-54343 come from?